static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/item.tsx

```diff
@@ -181,8 +194,8 @@ const Columns = styled('div')`
   display: flex;
   align-items: center;
   flex-shrink: 0;
-  min-width: 300px;
-  width: 300px;
+  min-width: 350px;
+  width: 350px;
```

Comment:
if I don't do this, the `Would Group` column would be too long. if there's another way to fix this, or if this should be behind the flag as well, please let me know.

Reply:
nobody looks at this page